upgrade_tests/helpers/common.py

```diff
@@ -0,0 +1,52 @@
+"""Common helper functions to run upgrade existence and scenario tests
+"""
+import os
+import pytest
+
+cur_ver = os.environ.get('FROM_VERSION')
+
+
+class VersionException(Exception):
+    """Version Exception if wrong satellite version provided"""
+    pass
+
+
+def run_to_upgrade(version):
+    """Decorator on test definition to run that test on given 'version' only.
```

Comment:
Docstring states this is a decorator. Decorator which receive parameter, like this one, must define a function which will receive decorated function as parameter. Besides that I was not able to find such a function inside of the body of this decorator. So I ask: have you decorated some test function and tested decorator?